examples/03_connectivity/plot_extract_regions_dictlearning_maps.py

```diff
@@ -114,7 +119,7 @@
 
 ###############################################################################
 # Plot resulting connectomes
-# ----------------------------
+# --------------------------
```

Comment:
This line does not render well...

Reply:
What do you mean? 
It seems to render fine in the artifacts:

![ex](https://user-images.githubusercontent.com/2639645/143261482-2be8b6d2-5711-46c2-8749-2a21de5e1eac.png)

Reply:
OK sorry, the lines do not render greatly on https://8786-1235740-gh.circle-artifacts.com/0/dev/connectivity/region_extraction.html but maybe this simply comes from the rst rendering on this particular page.